python/src/trezorlib/messages.py

```
@@ -1161,6 +1169,148 @@ def __init__(
+        self.address = address
+
+
+class BenfenSignTx(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = 12203
+    FIELDS = {
+        1: protobuf.Field("address_n", "uint32", repeated=True, required=False),
+        2: protobuf.Field("raw_tx", "bytes", repeated=False, required=True),
+        3: protobuf.Field("data_initial_chunk", "bytes", repeated=False, required=False),
+        4: protobuf.Field("coin_type", "bytes", repeated=False, required=False),
+        5: protobuf.Field("data_length", "uint32", repeated=False, required=False),
+    }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)